diff --git a/lisp/net/tramp-gvfs.el b/lisp/net/tramp-gvfs.el
index c4ec1121da..44b3ca35d5 100644
--- a/lisp/net/tramp-gvfs.el
+++ b/lisp/net/tramp-gvfs.el
@@ -125,6 +125,7 @@
 	 (or ;; Until Emacs 25, `process-attributes' could crash Emacs
 	     ;; for some processes.  Better we don't check.
 	     (<= emacs-major-version 25)
+	     (tramp-process-running-p ".gvfsd-fuse-wrapped")
 	     (tramp-process-running-p "gvfs-fuse-daemon")
 	     (tramp-process-running-p "gvfsd-fuse"))))
   "Non-nil when GVFS is available.")
